virttest/libvirt_xml/nodedev_xml.py

```diff
@@ -162,6 +181,10 @@ def __init__(self, virsh_instance=base.virsh):
                                  parent_xpath='/capability',
                                  marshal_from=self.marshal_from_address,
                                  marshal_to=self.marshal_to_address)
+        accessors.XMLElementList('iommuGroup_number', self,
+                                 parent_xpath='/iommuGroup',
+                                 marshal_from=self.marshal_from_address,
+                                 marshal_to=self.marshal_to_address)
```

Comment:
The iommuGroup_number should be an attribute? And here should be the 'iommuGroup_address' ?

Reply:
@kylazhang Updated, please have a check